service/src/main/resources/application.yml

```diff
@@ -132,4 +132,6 @@ drshub:
       mTlsConfig:
         keyPath: rendered/ras-mtls-client.key
         certPath: rendered/ras-mtls-client.crt
+  signedUrlDuration: 1h
```

Comment:
We should put this in the README and `openapi.yml` for this method so that users know how long the signed urls will work for.